src/dmd/id.d

```diff
@@ -86,6 +86,7 @@ immutable Msgtable[] msgtable =
     { "capture", "__capture" },
     { "this2", "__this" },
     { "_init", "init" },
+    { "initializer_", "initializer" },
```

Comment:
why the _ at the end?

Reply:
To avoid the conflict with the `initializer` function defined at module scope.